modules/api/current.js

```diff
@@ -121,19 +121,21 @@ module.exports = {
     const { id, testCaseId, attempt } = data;
     this.storage.setTestCaseStartedId(id, testCaseId);
     const { pickleId, isRetry: isTestCaseRetried } = this.storage.getTestCase(testCaseId);
+
     const {
       uri: pickleFeatureUri,
       astNodeIds: [scenarioId, parametersId],
     } = this.storage.getPickle(pickleId);
     const currentFeatureUri = this.storage.getCurrentFeatureUri();
     const feature = this.storage.getFeature(pickleFeatureUri);
     const launchTempId = this.storage.getLaunchTempId();
-
+    const isFirstFeatureInLaunch = currentFeatureUri === null;
```

Comment:
I suggest to move `const isFirstFeatureInLaunch = currentFeatureUri === null;` inside of `if (isNeedToStartFeature) {` condition block.